### PR TITLE
Add desmos testnet config

### PR DESCRIPTION
**Pull request by giorgionocera**
Adding the Desmos "morpheus-apollo-3" testnet configurations.

### CODE REVIEW

File: testnets/desmostestnet/chain.json [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "udaric"
+      }
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://github.com/desmos-labs/desmos",
+    "recommended_version": "v5.0.0",
```
Comment (JeremyParish69) [inline testnets/desmostestnet/chain.json:35]: based on the versions list, shouldn't this (and the compatible_versions list) be v5.2.0 ?

Comment (giorgionocera) [reply testnets/desmostestnet/chain.json:35]: It seems that on the testnet, the [upgrade proposal to v5.2.0](https://testnet.bigdipper.live/desmos/it/proposals/47) did not pass: https://testnet.bigdipper.live/desmos/it/proposals.
What do you suggest to do in this scenario?

Comment (JeremyParish69) [reply testnets/desmostestnet/chain.json:35]: In this case I'd suggest removing v5.2.0 object from the versions array, and this 'recommended version should be whatever the current version is (v5.1.0?)

Comment (giorgionocera) [reply testnets/desmostestnet/chain.json:35]: Okay, that sounds very reasonable. I'll provide the correct implementation 🚀

Comment (giorgionocera) [reply testnets/desmostestnet/chain.json]: I provided the modification from your suggestion following the previous version of the desmos (not demostestnet) configuration file (including the v5.2.0) [here](https://github.com/cosmos/chain-registry/blob/b26c994570f11209b8f6f1155d53e3610fc2f47a/desmos/chain.json).
Do you think it is okay now?

**Pull request merged**